genai/live/live_websocket_audiogen_with_txt.py

```diff
@@ -77,7 +75,9 @@ async def generate_content() -> str:
         },
     }
 
-    async with connect(WEBSOCKET_SERVICE_URL, additional_headers=headers) as websocket_session:
+    async with connect(
```

Comment:
Kindly fix this or rollback these formatting changes.

In Like 69, the line was allowed to be full but not this and following lines.